src/libs/freetype/lv_freetype_private.h

```diff
@@ -60,24 +60,43 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
-typedef struct _lv_freetype_cache_context_t lv_freetype_cache_context_t;
 typedef struct _lv_freetype_cache_node_t lv_freetype_cache_node_t;
 
+struct _lv_freetype_cache_node_t {
+    const char * pathname;
```

Comment:
If I understand correctly, the cache node stores glyph data. Do we need to store pathname here?

Reply:
not only for glyph data, but also glyph and draw data cache